calblueprint/meepanyar
# Schema update and new patch

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

Schema updated to add
1. Column in Tariff Plan to keep track of number of customers attached to tariff plan
2. Column in Tariff Plan to keep track of what kinds of meter readings this tariff plan can apply for
3. Adding a column to Customers to keep track of the last time its starting meter was changed
4. Active status for users
5. Updated Quantity number column for Purchase Requests

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## How to review
1. Check Airtable to make sure proper columns created
2. Go through patch workflow and make sure it works without conflict
3. Make sure the app runs so no type errors

[//]: # 'The order in which to review files and what to expect when testing locally'

## Relevant Links

### Online sources

[//]: # 'Optional - copy links to any tutorial or documentation that was useful to you when working on this PR'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @julianrkung

[//]: # 'This tags Julian as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
